_sass/globals/grid.scss

```diff
@@ -7,6 +7,11 @@
   overflow-x: hidden;
   overflow-y: auto;
 
+
+  &.overflow-y-hidden {
```

Comment:
This is for the mobile size, it keeps the sidebar from scrolling when the popover is active.